.gitignore

```diff
@@ -20,7 +20,11 @@ CMakeFiles/
 # Test exceptions
 test/UnitTests/*/results/
 test/LearningTest/*/*/*/results/
+test/*/results/
+test/LearningTest/*/*/*/results.ref/time.log
+results/
```

Comment:
I think line 23 and 25 are redundant with 21 and 22. We can choose to set only line 25 over 21 22 and 23 (to discuss)
line 24 is wrong, we have to keep time.log in results.ref

Reply:
Ok

Reply:
Just not to have a range too large how about `test/**/results/` ?